sql/rpl_slave.cc

```diff
@@ -7563,57 +7473,22 @@ wsrep_restart_point :
             Next iteration reads the same event. */
         break;
 
-<<<<<<< HEAD
       case SLAVE_APPLY_EVENT_AND_UPDATE_POS_APPLY_ERROR:
         /** fall through */
       case SLAVE_APPLY_EVENT_AND_UPDATE_POS_UPDATE_POS_ERROR:
         /** fall through */
       case SLAVE_APPLY_EVENT_AND_UPDATE_POS_APPEND_JOB_ERROR:
         main_loop_error = true;
         break;
-||||||| 35582423e36
-    while (!main_loop_error && !sql_slave_killed(thd, rli)) {
-      Log_event *ev = nullptr;
-      THD_STAGE_INFO(thd, stage_reading_event_from_the_relay_log);
-      DBUG_ASSERT(rli->info_thd == thd);
-      THD_CHECK_SENTRY(thd);
-      if (saved_skip && rli->slave_skip_counter == 0) {
-        LogErr(INFORMATION_LEVEL, ER_RPL_SLAVE_SKIP_COUNTER_EXECUTED,
-               (ulong)saved_skip, saved_log_name, (ulong)saved_log_pos,
-               saved_master_log_name, (ulong)saved_master_log_pos,
-               rli->get_group_relay_log_name(),
-               (ulong)rli->get_group_relay_log_pos(),
-               rli->get_group_master_log_name(),
-               (ulong)rli->get_group_master_log_pos());
-        saved_skip = 0;
-      }
-=======
-    while (!main_loop_error && !sql_slave_killed(thd, rli)) {
-      Log_event *ev = nullptr;
-      THD_STAGE_INFO(thd, stage_reading_event_from_the_relay_log);
-      assert(rli->info_thd == thd);
-      THD_CHECK_SENTRY(thd);
-      if (saved_skip && rli->slave_skip_counter == 0) {
-        LogErr(INFORMATION_LEVEL, ER_RPL_SLAVE_SKIP_COUNTER_EXECUTED,
-               (ulong)saved_skip, saved_log_name, (ulong)saved_log_pos,
-               saved_master_log_name, (ulong)saved_master_log_pos,
-               rli->get_group_relay_log_name(),
-               (ulong)rli->get_group_relay_log_pos(),
-               rli->get_group_master_log_name(),
-               (ulong)rli->get_group_master_log_pos());
-        saved_skip = 0;
-      }
```

Comment:
Missing block??

Reply:
It's not missing, I kept the pxc version of the diff.

Reply:
You're right! This block is around 50 above the conflict. It appeared to me that it is missing. Thanks for clarifying.